### PR TITLE
Add TCK process

### PR DESCRIPTION
**Pull request by ivargrimstad**
Signed-off-by: Ivar Grimstad <ivar.grimstad@eclipse-foundation.org>

### CODE REVIEW

File: content/committees/specification/TCKProcess.md [hunk 1]
```diff
@@ -0,0 +1,412 @@
+---
+title: "akarta EE TCK Process 1.0
```
Comment (chrisguindon) [inline content/committees/specification/TCKProcess.md]: There is a typo in the title and we can include a date value?

For example: 
date: '2017-09-26T14:02:37+02:00'

**Comment by chrisguindon** [comment]
I also noticed that this PR is not for the right branch:

ivargrimstad wants to merge 1 commit into jakartaee:master

The website source is on the src branch

File: content/committees/specification/TCKProcess.md (hunk 1 above)
Comment (ivargrimstad) [reply content/committees/specification/TCKProcess.md]: I fixed the typo and added today's date.

**Comment by ivargrimstad** [comment]
@chrisguindon  Can I change the target branch of this PR? Or close it and submit another?

**Comment by chrisguindon** [comment]
You can change it but makes your PR invalid since the history on both branch do not match:
https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-base-branch-of-a-pull-request

Can you create a new one?

**Pull request closed without merging**